modules/core/src/main/java/org/apache/ignite/internal/processors/security/IgniteSecurityProcessor.java

```diff
@@ -118,8 +130,13 @@ public IgniteSecurityProcessor(GridKernalContext ctx, GridSecurityProcessor secP
             uuid -> nodeSecurityContext(marsh, U.resolveClassLoader(ctx.config()), node))
             : secPrc.securityContext(subjId);
 
-        if (res == null)
-            throw new IllegalStateException("Failed to find security context for subject with given ID : " + subjId);
+        if (res == null) {
+            log.warning("Switched to the 'deny all' policy because of failing to find a security context [subjId=" + subjId + ']');
```

Comment:
Let's also add the type (node or client)

Reply:
OK, will do